command/agent/consul/client.go

```diff
@@ -345,6 +345,15 @@ INIT:
 				c.logger.Printf("[INFO] consul.sync: successfully updated services in Consul")
```

Comment:
Should this be a DEBUG?

Reply:
Ah, github's default hiding makes it appear noisier than it really is. It only logs when `failures > 0` which means it logs as an indication that Consul has recovered.

In a steady state it will _not_ log. I kind of like it because otherwise you would only see Consul syncing failures in the logs and not a clear success message.